.env.example

```diff
@@ -8,3 +8,4 @@ LOG_LEVEL=debug
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 # WEBHOOK_PROXY_URL=
 PERSPECTIVE_API_KEY=
+GITHUB_ACCESS_TOKEN=
```

Comment:
If this is optional, maybe comment it out in the example .env so that folks will know that its not necessary to run the application.

Reply:
No, it will be required.

For now it is just used in the sandbox but in the next PR it will be used in main app code.